### PR TITLE
Feauture/wa 52: Add new architecture of the blog

### PR DESCRIPTION
**Pull request by Davidalbort**
There were some changes in files to useDate, ListCardPost, linkNavigation for they can to adapt to new feature

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 *.yml
-sitemap.xml
```
Comment (MEGAFRAN) [inline .prettierignore:2]: why this was removed?

File: app/constants/types/components_props/types.ts [hunk 2]
```diff
@@ -352,3 +353,37 @@ export type QuestionnarieProps = {
 export type QuestionnarieFormState = {
   [key: string]: number
 }
+
+export type Ctx = {
```
Comment (MEGAFRAN) [inline app/constants/types/components_props/types.ts:357]: could this name be more descriptive?

File: app/hook/useDate.ts [hunk 3]
```diff
@@ -78,17 +77,24 @@ export default function useDate() {
     const dataString = `${day} ${monthLetters}, ${year}`
     return dataString
   }
-
-  const generateEndpointCardPost = ({ date, title }: GenerateEndPointCardPostParam): string => {
+  // old generation endpoint by cardPost
```
Comment (MEGAFRAN) [inline app/hook/useDate.ts:80]: is possible to remove all these comments?

File: lib/getStaticPostDetail.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+export async function getI18nProps({ ctx, ns = ["common"] }: GetI18Props) {
+  const locale = ctx?.params?.locale || i18nextConfig.i18n.defaultLocale
+  const props = {
+    ...(await serverSideTranslations(locale, ns)),
+  }
+  return props
+}
+
+export function makeStaticProps({ ns = [] }: MakeStaticProps) {
```
Comment (MEGAFRAN) [inline lib/getStaticPostDetail.ts:35]: what means 'ns'?

File: next.config.js [hunk 5]
```diff
@@ -5,6 +5,9 @@
 const nextConfig = {
   reactStrictMode: true,
   trailingSlash: true,
+  images: {
```
Comment (MEGAFRAN) [inline next.config.js:8]: could you explain me what does this?

File: app/constants/types/components_props/types.ts (hunk 2 above)
Comment (Davidalbort) [reply app/constants/types/components_props/types.ts:357]: Yes, ofcourse

File: app/hook/useDate.ts (hunk 3 above)
Comment (Davidalbort) [reply app/hook/useDate.ts:80]: Yes, ofcourse.

File: .prettierignore (hunk 1 above)
Comment (Davidalbort) [reply .prettierignore:2]: Sorry, I accidentally removed it

File: next.config.js (hunk 5 above)
Comment (Davidalbort) [reply next.config.js:8]: Yes, I added the image component of nextjs and then I had to add domain of the image, but not is neccesary at moment. Sorry.

**Pull request merged**